.github/workflows/tests.yml

```diff
@@ -1,44 +0,0 @@
-name: Build
```

Comment:
would be nice to have this file renamed to `workflow.yml` and then changed to keep history